private/buf/bufformat/formatter.go

```diff
@@ -1226,6 +1260,7 @@ func (f *formatter) writeBody(
 		f.Indent()
 		f.writeNode(closeBrace)
 		f.writeTrailingEndComments(info.TrailingComments())
+		f.SetPreviousNode(closeBrace)
```

Comment:
Another bug. This code path was unexercised with the old version protocompile -- again because the comment would usually have been attributed as a trailing comment to previous token, so we'd not fall into this condition.

But with the new protocompile, it popped up as a mysterious missing newline in between elements (all because some logic that decides to add a newline wants to know if the last thing was a closing brace). Other call paths through this function do set this correctly after writing it.